addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -252,6 +252,18 @@ options.Class.include({
         });
     },
 
+    setShadow(previewMode, widgetValue, params) {
+        let [shadowColor, shadowX, shadowY, shadowBlur, shadowSpread, shadowInset] = this._getShadowValues(this.$target);
+        if (widgetValue) {
+            shadowY = '8px';
+            shadowBlur = '16px';
+        } else {
+            shadowX = shadowY = shadowBlur = shadowSpread = '0px';
+        }
+        this.$target[0].style.setProperty('box-shadow',
+            `${shadowColor} ${shadowX} ${shadowY} ${shadowBlur} ${shadowSpread} ${shadowInset || ''}`, 'important');
```

Comment:
Sure that all supported browsers use the same order for the sub-properties? I actually thought color was last.

Reply:
I think the order doesn't matter for inset and color

Reply:
The order does not matter indeed, my question is: do all browsers **return** the properties in the same order?

Reply:
Chrome and firefox were fine but I just tested edge and color is indeed last

Reply:
Damn CSS convention :/ I guess it makes your task a bit less easy :/

Reply:
I can either make a fully custom options like in the first commit or have a condition in we-multi `setValue` based on `this._methodParams.cssProperty` so the way to split is defined on a css-property basis. What do you think?

Reply:
we-multi should be independent of any relation to css properties. I think you should instead patch the `computeWidgetState` of `selectStyle`, so that when the box-shadow css property is used it reorders the values so that we-multi works.

So:
- _computeWidgetState of selectStyle: get the css property value
- if box-shadow then split yourself in _computeWidgetState and re-join in the right order
- we-multi receives the value and (re)split a "correctly-ordered" value